Opserver/Views/SQL/Dashboard.Model.cs

```diff
@@ -13,7 +13,8 @@ public enum SQLViews
         Top = 5,
         Connections = 6,
         Databases = 7,
-        QueryStore = 8
+        QueryStore = 8,
+        BlitzCache = 9,
```

Comment:
This feels like it will be hassle when it comes to the community all working on similar extensions 😁 and wanting to merge.

Reply:
it was even a pain for me with branches. I just rememberd to make the blitzcache 9 when i made this branch